packages/zent-popover/src/trigger/HoverTrigger.js

```
@@ -161,7 +162,7 @@ function makeHoverLeaveRecognizer({ leaveDelay, onLeave, isOutSide }) {
             state.transit(HoverState.Started);
           }
         }
-      }
+      }, 149)
```

default show/hide delay is 150..

set it to 149 just for test... lol